client/src/scenes/Login/index.jsx

```diff
@@ -69,30 +71,39 @@ class Login extends Component {
   render() {
       const { isLoading, isEmailValid, isPasswordValid } = this.state;
       return !this.props.isAuthorized ? (
-          <Grid textAlign="center" verticalAlign="middle" className="fill">
-              <Grid.Column style={{ maxWidth: 450 }}>
-                  <Header as="h2" color="blue" textAlign="center">
+          <Grid textAlign="center" style={{ height: '100vh' }} verticalAlign="middle" className="login-grid">
+              <Grid.Column style={{ maxWidth: 400 }}>
```

Comment:
Are you sure that's no other way to set max-width?

Reply:
fixed